website/content/maintainers/templates/_index.md

```diff
@@ -4,50 +4,62 @@ linkTitle: "Templates"
 layout: single
 date: 2020-11-07
 description: >
-  Jump start your CNCF project by using our recommended templates
+  Jump start your CNCF project by using our templates
+aliases:
+- /maintainers/github/templates/
 ---
 
 We have a [project template] repository that has templates for everything your project needs
-to get started and eventually join the CNCF. You can either create a new repository using the repository template or copy just the templates that you require.
+to get started and eventually join the CNCF. 
 
 ## Getting Started
 
-1. Click **Use this template** and create a copy of this repository.
+The templates are stored in the [project-template] repo,
+but we also have how-to guides here on the website that are a better place to
+start if you haven't used a particular template before.
 
-    ![Green button that says "Use this template"](https://user-images.githubusercontent.com/1368985/95903529-e9c32f00-0d5b-11eb-8723-4369f7c9e044.png)
-1. Remove **.github/settings.yml**. This is not a template and contains
-   configuration specific our repository. You should not keep this file.
-1. Customize every [required file](#required-files) and address each TODO item.
-1. Security templates may be found in the [Security TAG's project resources
-   folder](https://github.com/cncf/tag-security/tree/main/project-resources).
+Please note that the security templates may be found in the
+[Security TAG's project resources folder](https://github.com/cncf/tag-security/tree/main/project-resources).
 
 ### Customize Templates
 
-Each file is a template with instructions to customize the contents for your project.
+Each template contains instructions to customize the contents for your project.
 Most files use comments with TODO to call out where you need to make changes. We recommend
 viewing the files in raw or text form so that you can see the comments. 
 
 For example in markdown files, we use `<!-- TODO: ... -->` to provide additional
 guidance or indicate where action is required but you won't see those comments
 when you view the markdown file in GitHub unless you view the raw text.
 
-## Required Files
+## How-To Guides
+
+* [CONTRIBUTING.md](contributing.md)
+* [Governance Introduction](governance-intro.md)
+* [GOVERNANCE-elections.md](governance-elections.md)
+* [GOVERNANCE-maintainer.md](governance-maintainer.md)
+* [GOVERNANCE-subprojects.md](governance-subprojects.md)
+* [MAINTAINERS.md](maintainers.md)
+* [REVIEWING.md](reviewing.md)
+* [Issue Labels](issue-labels.md)
+
+## Available Templates
 
 * [LICENSE](https://github.com/cncf/project-template/blob/main/LICENSE)
```

Comment:
Would it be helpful to sort the templates by their file name so that it matches the order that it's displayed in GitHub when viewing the template repo? Also just to help people scan for a particular template.